source/community.applications/usr/local/emhttp/plugins/community.applications/skins/Narrow/skin.html

```
@@ -112,6 +112,36 @@
+<!-- Loading placeholder cloned into #caDiffContent while the feed JSON is
+     being downloaded / parsed. The `.caDownloadProgress` <div> below is the
+     same class the page-level nchan subscriber (Apps.page) targets — it sets
+     .html(message), and after 5s of activity it removes .ca_hide, so progress
+     text shows up inside this overlay without any z-index gymnastics.
+     Wrapped in .caDiffLoadingBox so flex auto-margins center the content
+     block inside the flex-column #caDiffContent. -->
+<div class='ca_hide'>
+	<div id='caDiffLoading'>
+		<div class='caDiffLoadingBox'>
+			<div class='ca_center caLogoIcon'></div>
+			<div class='ca_center ca_italic'><?tr("Downloading application feed and rendering diff...")?></div>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)